packages/blade/src/components/IconButton/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './IconButton';
```

Comment:
Named export? 😅 
```suggestion
export { IconButton, IconButtonProps } from './IconButton';
```

Reply:
Oh, are we doing this for all components now?

Reply:
I think so, yes 😅

Reply:
Done in https://github.com/razorpay/blade/pull/613/commits/aaf6244a6828a7dd0e0c8278c201e6ab43ced2e1